lib/build.gradle.kts

```diff
@@ -71,16 +71,66 @@ afterEvaluate {
     }
 }
 
-dependencies {
-    implementation(platform(libs.androidx.compose.bom))
-    implementation(libs.androidx.compose.material)
-    implementation(libs.androidx.compose.ui.ui)
-    implementation(libs.androidx.compose.ui.tooling.preview)
+kotlin {
 
-    debugImplementation(libs.androidx.compose.ui.test.manifest)
-    debugImplementation(libs.androidx.compose.ui.tooling)
+    val isAndroidApp = plugins.hasPlugin("com.android.application")
+    val isAndroidLibrary = plugins.hasPlugin("com.android.library")
+    if (isAndroidApp || isAndroidLibrary) {
```

Comment:
The lib module will never be a android app module. Is isAndroidApp necessary?

Reply:
you´re correct. it can safely removed!